### PR TITLE
build: add target configuration of "riscv64imac-unknown-none-elf"

### PR DESCRIPTION
**Pull request by ZhengLongBing**
- This addition facilitates better integration and usage with IDEs like VSCode and RustRover.

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[build]
+target = "riscv64imac-unknown-none-elf"
```
Comment (luojia65) [inline .cargo/config.toml]:
```suggestion
[build]
target = "riscv64imac-unknown-none-elf"

```
There should be a new line at the end of each code file, specifically `.toml` and `.rs` files.

**Review by luojia65 — CHANGES_REQUESTED** [review]

File: .cargo/config.toml (hunk 1 above)
Comment (ZhengLongBing) [reply .cargo/config.toml]: Thank you for your reminder, I guarantee that I will not commit anymore in the future.

**Review by luojia65 — APPROVED** [review]
This change LGTM. r? @Placebo27

**Review by Placebo27 — CHANGES_REQUESTED** [review]
LGTM! Thanks for your target configuration.

But it is recommended to use `build` instead of `config.toml` as the header.  And `build` is more suitable for modifying project build system.
For more accurate expression, you can change the header to `build: add target configuration of "riscv64imac-unknown-none-elf"`.

**Comment by ZhengLongBing** [comment]
Thank you very much for your suggestion!

**Review by Placebo27 — APPROVED** [review]

**Comment by Placebo27** [comment]
You may forget to change this commit message  for accurate expression. Hope you will pay more attention in your future contribution.

**Pull request merged**